shimmer/learn/governance/governance-tech.md

```diff
@@ -0,0 +1,128 @@
+- **Voting Open:** The Voting Open stage lasts seven days and starts when a predefined milestone in the Shimmer network is reached. After the network has passed this milestone, users can cast, change, revoke, or redo the vote using the governance functionality of the Shimmer Firefly wallet. It's important to note that changes of opinion during this timeframe will not affect the final vote count. Ideally, voters should cast their votes before the next stage begins to ensure the highest possible vote count.
+- **Counting Start:** This directly follows the Voting Open stage. The node software counts the user's vote, which gains weight with increasing holding time at every milestone passed (a milestone is issued every five seconds). If the voter holds their vote for the predefined number of milestones, the voter will reach 100% of the possible votes related to their voting power (i.e. the number of tokens they have used in the vote). If users change their vote in this stage, the participation plugin updates these changes from the time when they were made. However, the previously counted votes are still valid and do not change, even if the voter’s opinion or voting power changes (examples are provided below for clarification).
+- **Counting Stops:** The accumulation of votes ends. All nodes will now be able to produce and verify the final result of the vote.
+
+### Voting details
+
+The voting mechanism utilizes Shimmer’s ability to issue value transactions with attached metadata. 
+To participate in voting, the user must define the amount of SMR tokens that should be used for governance votes, which represents their voting power, 
+for each wallet associated with their Shimmer Firefly profile. 
+The tokens that represent the voting power will be sent to a new output (UTXO) at the same address (i.e., the user sends the tokens to themselves) using the 
+Manage Voting Power function in the new Governance section of Shimmer Firefly.
+
+This Governance Output to which the tokens are allocated will now be used for all governance votes of this wallet. 
+The voter can decide to vote on all kinds of governance questions with the allocated voting weight.
+Whenever a vote is cast, it uses the tokens that are stored in the Governance Output for a transaction 
+(again, the user sends the tokens to themselves, to the same output, using the same address). 
+
+During this transaction, metadata that includes the ballot event ID, question ID, and answer will be attached to the UTXO where the tokens reside. 
+This immutably writes the metadata onto the UTXOs of this address in the Tangle. So, after the voting transaction is complete, 
+the tokens will still reside at the same address as before but now they will have attached metadata representing the voter's opinion for the referendum.
+
+:::👉Tip: 
```

Comment:
The emoticon doesn't render well, but you can add a tip title if you want, for example 
```suggestion
:::tip Storage Deposits
```